src/api/operator/operatorApi.ts

```diff
@@ -197,12 +197,19 @@ export async function getOrder(params: GetOrderParams): Promise<RawOrder | null>
 export async function getOrders(params: GetOrdersParams): Promise<RawOrder[]> {
   const { networkId, ...searchParams } = params
   const { owner, sellToken, buyToken } = searchParams
+  const oneYearAgo = (): string => {
+    const currentDate = new Date()
+    const yearAgo = new Date(currentDate.setMonth(currentDate.getMonth() - 1))
+
+    return String(Math.floor(yearAgo.getTime() / 1000))
+  }
```

Comment:
We probably won't need this function, but anyway, it doesn't seem to do what I think it should:
![screenshot_2021-08-31_14-38-14](https://user-images.githubusercontent.com/43217/131579063-4948ec21-3277-49bc-8444-75b4702842b8.png)

Also, we have https://date-fns.org/ as a dependency, makes our lives easier :)